tsconfig.json

```diff
@@ -7,15 +7,11 @@
     "moduleResolution": "node",
     "target": "es2022",
     "lib": ["es2022"],
-    "esModuleInterop": true,
-    "declaration": true, 
-    "declarationDir": "./build", 
-    "outDir": "./build",
```

Comment:
this was actually creating a dupicate build folder outside dist. not necessary anymore now that tsup and rullup can run at the same time